### ISSUE TITLE
HDMI-CEC problem or is it my TV?

### ISSUE THREAD

**Issue by Haris1977** [OpenELEC/OpenELEC.tv#4049]
Greetings

I have made it possible to use my Philips tv remote control (through universal mode) to control my HTS (lg home theater) along with OE 5.06. 

But my strange problem is that while i am in OE, i m able to hit HTS button (green square in photo) and i can control my Lg Home Theater from my remote. But..when i hit back the TV button (red square in photo) to continue using kodi with hdmi-cec, i cannt control it with my remote anymore..:(

Look photo here: http://www.2shared.com/photo/p1PrfyNa/20150328_130439.html

Only if i close my HTS, OE will work again (with hdmi cec).

So, in other words, only if i power off my HTS, kodi's hdmi-cec will work again !!:(

So is there anything i can do to make it work again?? Is this my TV's problem or kodi's hdmi cec problem??

Any thoughts appreciated!

My machines: Rpi, Philips TV 55 inch, Lg Home theater, latest openelec build

**Comment by Haris1977** [comment]
I found where the problem relies:

The problem is on the hdmi port (or) cable of my LG Home Theater set. If i remove it, all work fine..
